.github/workflows/nightly-release.yml

```diff
@@ -39,11 +39,13 @@ jobs:
           while git tag | grep ${TAG_BASE}${INDEX}; do
               ((INDEX+=1))
           done
+          git submodule update --remote hw-latest/caliptra-rtl
+          CHECK_RELEASE_SYNC=$(git status -s | head -1)"
```

Comment:
git status -s is for humans, not scripts (and may change to always print something in the future). Use --porcelain instead